component/src/main/java/org/wso2/extension/siddhi/io/wso2event/sink/WSO2EventSink.java

```diff
@@ -18,6 +18,20 @@
 
 package org.wso2.extension.siddhi.io.wso2event.sink;
 
+import io.siddhi.annotation.Example;
+import io.siddhi.annotation.Extension;
+import io.siddhi.annotation.Parameter;
+import io.siddhi.annotation.util.DataType;
+import io.siddhi.core.config.SiddhiAppContext;
+import io.siddhi.core.exception.ConnectionUnavailableException;
+import io.siddhi.core.stream.ServiceDeploymentInfo;
+import io.siddhi.core.stream.output.sink.Sink;
+import io.siddhi.core.util.config.ConfigReader;
+import io.siddhi.core.util.snapshot.state.State;
+import io.siddhi.core.util.snapshot.state.StateFactory;
+import io.siddhi.core.util.transport.DynamicOptions;
+import io.siddhi.core.util.transport.OptionHolder;
+import io.siddhi.query.api.definition.StreamDefinition;
```

Comment:
Import statements are usually sorted so that most fundamental packages come first. IMO org.apache.log4j.Logger etc should come before io.siddhi packages.

Reply:
According to WSO2 checkstyles guidelines[1], these imports should be ordered according to alphabetical order. If we did not follow that a checkstyle error occurs when building the package.

ex:
`/Users/raveen/WSO2/Public/Siddhi-Extensions/siddhi-io-wso2event/component/src/main/java/org/wso2/extension/siddhi/io/wso2event/sink/WSO2EventSink.java:22: error: Wrong order for 'io.siddhi.annotation.Example' import.
`

[1] https://github.com/wso2/code-quality-tools/blob/master/checkstyle/checkstyle.xml#L97